src/celeritas/io/ImportMuPairProductionTable.hh

```diff
@@ -0,0 +1,45 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/io/ImportMuPairProductionTable.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <vector>
+
+#include "ImportPhysicsVector.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Sampling table for electron-positron pair production by muons.
+ *
+ * This 3-dimensional table is used to sample the energy transfer to the
+ * electron-positron pair, \f$ \epsilon_p \f$. The outer grid stores the atomic
+ * number using 5 equally spaced points in \f$ \log Z \f$; the x grid tabulates
+ * the ratio \f$ \log \epsilon_p / T \f$, where \f$ T \f$ is the incident muon
+ * energy; the y grid stores the incident muon energy using equal spacing in
+ * \f$ \log T \f$.
+ */
+struct ImportMuPairProductionTable
+{
+    //!@{
+    //! \name Type aliases
+    using ZInt = int;
+    //!@}
+
+    std::vector<ZInt> atomic_number;
+    std::vector<ImportPhysics2DVector> physics_vectors;
+
+    explicit operator bool() const
+    {
+        return !atomic_number.empty()
+               && physics_vectors.size() == atomic_number.size();
+    }
+};
```

Comment:
FWIW the import SB table is kept at a high level as `map<int, table>`, which makes it compatible with `celeritas/io/ImportedElementalMapLoader.hh` and lets us choose to read it from a file *or* from the imported data. I think we should have the same approach for both. Perhaps keep the comment but use:
```suggestion
using ImportMuPairProductionTable = ImportPhysics2DVector;
```
since the SBTable is for a single element? Then 
```c++
    using ImportMuPPMap = std::map<ZInt, ImportMuPairProductionTable >;
```
in the IO data.

Reply:
Sure... it's a bit different from the other data because it can _only_ be imported from the model, not read from a file, and interpolates on a fixed Z grid rather than building the tables for each element in the problem, but we can do that.

Reply:
Can you explain what you mean by interpolating on fixed Z? As in, the model hardcodes a few different Z values that don't depend on the materials in the problem, and linearly interpolates on that? In that case yeah I agree let's keep the layout as-is (but still replace ImportSBTable with ImportPhysics2DVector).

Reply:
Yeah, exactly. From the PRM:
> a constant increment in ln𝑍 is chosen. The shape of the sampling distribution does depend on 𝑍, but very weakly, so that eight points in the range 1 ≤ 𝑍 ≤ 128 are sufficient

(although in the code there are only 5 points: `zdat[5] = {1, 4, 13, 29, 92}`).

Reply:
Might be worth going back to the original source for the table since they might have the full Z... also I wonder if we should be only importing Z values based on the range of materials in the problem (or does it matter...)

Reply:
They're constructed in the model from the differential cross section, which is calculated from a formula rather than tables. Maybe someday we can implement the cross section calculator and build the tables ourselves...

True, we could restrict the table based on the elements in the problem... I can add that if you think it's worth it.

Reply:
Building the cross sections sounds like a better investment of our time, and there's plenty of stuff better than building cross sections 😅